src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -29,16 +29,19 @@ internal class SilentRequest : RequestBase
             _silentParameters = silentParameters;
         }
 
-        private async Task<IAccount> GetSingleAccountForLoginHintAsync(string loginHint)
+        private async Task<IAccount> GetSingleAccountForLoginHintAsync(string loginHint, bool singleAccount = false)
```

Comment:
do you need to add this? don't you have this in the _silentParameters?

Reply:
not needed. its dead code